Docs/articles/Kernel/CGS.md

```diff
@@ -78,7 +80,7 @@ namespace GraphicTest
         {
             try
             {
-                Pen pen = new Pen();
+                Color col = new Color();
```

Comment:
You can't create an instance of Color

Reply:
better remove it, because that variable is not used

Reply:
Yeah, just rm it